### ISSUE TITLE
How to Pass Global Headers to SubGraph subscription in GraphQl Federation

### ISSUE THREAD

**Issue by myworkfoce** [TykTechnologies/tyk#4343]
Currently, global headers are passed from supergraph to subgraph.  However, subgraph subscriptions don't receive the same global headers when using websocket connection. 

I suppose global headers should behave the same way in subscriptions as they are behaving in regular queries. Is there a different way to do it or is it not supported yet?

**Comment by jay-deshmukh** [comment]
yes to pass headers for subscription requests, `Allow query parameter as well as header` needs to be enabled and header needs to be bass as query param
